client/components/global/ChangePasswordForm.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useContext } from 'react';
+import { BrowserRouter as Router, Switch, Route, Link, useParams } from 'react-router-dom';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import axios from 'axios';
+
+export const ChangePasswordForm = () => {
+  let user = useParams();
+  const [show, setShow] = useState(false);
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [err, setErr] = useState(null);
+  const [noMatch, setNoMatch] = useState(false);
+  const [response, setResponse] = useState(null);
+
+  const handleChangePassword = () => {
+    axios
+      .post(`/reset/${user.email}/${user.token}/${password}`)
+      .then((res) => {
+        setResponse(res.data);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
+  if (response) {
```

Comment:
This conditional render block is a bit awkward, you should try to have one return block within a component